### PR TITLE
added code

### PR DESCRIPTION
**Pull request by Szpytowski**
Added my code, I struggled with margin and padding on this one.

### CODE REVIEW

**Review by Solnick — CHANGES_REQUESTED** [review]
image
There are some obvious differences to fix, for example the font.

**Review by Solnick — CHANGES_REQUESTED** [review]
image
Also make sure to fulfill all the requirements

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
+
+body{
```
Comment (Solnick) [inline css/style.css]:
```suggestion
body {
```
Remember about space after the selector.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    background-color: #f5f5f5;
+    margin: 0;
+}
+
+.ticket{
+    box-sizing: border-box;
```
Comment (Solnick) [inline css/style.css]: use box-sizing globally, for example using `body` or `*` selector

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    width: 782px;
+    height: 273px;
+    background-color: #efc684;
+    border-radius: 10px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-family: Arial, sans-serif;
```
Comment (Solnick) [inline css/style.css]: Please find the correct font in figma 
image

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+    position: relative;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.2);
+}
+
+.popcorn img{
+    width: 134.26px;
```
Comment (Solnick) [inline css/style.css]: don't use decimal point in pixels

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+    margin-right: 41px;
+    transform: rotate(1.62deg);
```
Comment (Solnick) [inline css/style.css]: why such a value?

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+.left-section{
+    width: 45%;
+}
+
+h1{
+    margin: 0;
+    padding: 0;
+    font-family: 'Roboto';
```
Comment (Solnick) [inline css/style.css]: remember to use the fallback font
```
    font-family: 'Roboto', sans-serif;
```

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+}
+
+.ticket-type{
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+}
+
+h2{
+    display: flex;
```
Comment (Solnick) [inline css/style.css]: Do you really want to add those styles for every h2 on the web page? Please us the class selector to be more specific. Using the tag selectors is rarely needed and should be used only when you really want to globally change all the elements in the webpage.

File: css/style.css [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+.middle-section{
+    margin-left: 45px;
+    margin-bottom: 15px;
+}
+
+.barcode{
+    margin-top: 45px;
+}
+
+
```
Comment (Solnick) [inline css/style.css]: When programming leave only one blank line at the end of any file

File: index.html [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+        <div class="left-section">
+            <h1>CINEMA TICKET</h1>
+            <div class="ticket-type and price">
+                <h2 class="vip">VIP</h2>
+                <h2 class="price">$10</h2>
+            </div>
+        </div>
+        <div class="middle-section">
+            <p>08:00 PM</p>
+            <p>GATE: 03 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; SEAT: B43</p>
```
Comment (Solnick) [inline index.html]: Please don't use the space sign for positioning, instead use CSS rules, for example flexbox

**Review by Solnick — CHANGES_REQUESTED** [review]

File: css/style.css [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+}
+
+.popcorn img {
+    width: 130px;
+    transform: rotate(1.62deg);
+    transition: transform 0.3s ease-in-out;
+}
+
+.popcorn-container:hover .popcorn img {
+    transform:  rotate(5deg);
```
Comment (Solnick) [inline css/style.css]:
```suggestion
    transform: rotate(5deg);
```
One space is enough

File: css/style.css [hunk 11]
```diff
@@ -0,0 +1,142 @@
...
+.little-popcorns  img {
+    width: 20px;
+    height: auto;
+    margin-bottom: 15px;
+    opacity: 0;
+    transition: 0.3s ease-in-out;
+}
+
+.popcorn-container:hover img {
+    opacity: 1;
```
Comment (Solnick) [inline css/style.css]: That's a good solution. I just want to let you know that you could've hide the little popcorn images under the popcorn box by using `transform: translate(x, y);` then the transition would make it pop out of the popcorn box. You can try it if you wish.

**Review by Solnick — APPROVED** [review]

File: css/style.css (hunk 11 above)
Comment (Szpytowski) [reply css/style.css]: done

File: css/style.css (hunk 10 above)
Comment (Szpytowski) [reply css/style.css]: done

**Pull request merged**